### PR TITLE
added beginners-contributing-guide.md

### PR DESCRIPTION
**Pull request by WhiteNight121**
added the beginners-contributing-guide.md i was not sure what to name it hope its what everyone is looking for.

### CODE REVIEW

File: docs/basics/getting-started.md [hunk 1]
```diff
@@ -16,3 +16,4 @@ Things here might include:
 ### Software
   * [docs.linuxserver.io](https://docs.linuxserver.io/)
   * [Docker](https://www.docker.com/) & [Docker Compose](https://docs.docker.com/compose/)
+  * [Plex](https://www.plex.tv/)
```
Comment (py-crash) [inline docs/basics/getting-started.md:19]: This change should not be part of this PR

File: docs/basics/beginners-contributing-guide.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+-  Verify you are in the wiki directory.
+
+- Type `git fetch upstream` to fetch the branches and their respective commits from the upstream repository
+
+- Verify the "dev" branch is checked out type `git checkout -b dev`
+
+- git merge upstream/dev
+
+## Recommended Reading (watching)
```
Comment (py-crash) [inline docs/basics/beginners-contributing-guide.md:111]: Shouldn't be Reading/watching

File: docs/basics/beginners-contributing-guide.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+* This guide was written for linux
+
+__Note__: **This document will use variables thought out make sure to change them as needed.**
+
+### 1. Install Git
+This section will install Git.
+
+0. Log in to the server you are going to use for Git.
+
+0. Type `add-apt-repository ppa:git-core/ppa`.
```
Comment (py-crash) [inline docs/basics/beginners-contributing-guide.md:15]: You can just install git from the repos, don't need to add a ppa and pollute your installation.

File: docs/basics/beginners-contributing-guide.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+0. Navigate to the [upstream](https://github.com/selfhostedshow/wiki).
+
+0. Click ![Fork](images/fork.PNG) in the top right to copy the "wiki" repository to your Github account.
+
+0. Log in to the server you are going to use for Git and `cd` to the directory you want to download the wiki repository to.
+
+0. Type `git clone https://github.com/YOURGITHUBSUSERNAME/wiki && cd wiki`.
+
+0. Verify the "dev" branch is checked out type `git checkout -b dev`.
```
Comment (py-crash) [inline docs/basics/beginners-contributing-guide.md:34]: This one should be `git checkout dev`, because a `dev` branch already exists.

File: docs/basics/beginners-contributing-guide.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+0. Type `git add YOURFILENAME1.md YOURFILENAME2.md` to add the changed files to your commit.
+
+0. Type `git commit -m 'updated x,y,z'` to commit the changes to your fork with comments.
+
+0. Type  `git push origin dev` to push your commit to your fork.
+
+0. Type your username for github:
+
+0. Type your password for github:
```
Comment (py-crash) [inline docs/basics/beginners-contributing-guide.md:83]: These are sentences, should en with a period (`.`) not `:`.

File: docs/basics/beginners-contributing-guide.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+0. The commit has now been added to the pull request the Wiki-Admins will review and if approved will get pushed to the master branch for publishing.
+
+### 2 Syncing a Fork
+This Section will "pull" from the [upstream](https://github.com/selfhostedshow/wiki) repository to your local repository. **This should be done every time before you make any changes to your local wiki in order to make sure everything is in sync**.
+
+-  Verify you are in the wiki directory.
+
+- Type `git fetch upstream` to fetch the branches and their respective commits from the upstream repository
+
+- Verify the "dev" branch is checked out type `git checkout -b dev`
```
Comment (py-crash) [inline docs/basics/beginners-contributing-guide.md:107]: The same. Should be `git checkout dev`, because this one will give a fatal error

File: docs/basics/beginners-contributing-guide.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+* Docker is already running in your environment
+* You have a [Github](https://github.com) account and are logged in
+* This guide was written for linux
+
+__Note__: **This document will use variables thought out make sure to change them as needed.**
+
+### 1. Install Git
+This section will install Git.
+
+0. Log in to the server you are going to use for Git.
```
Comment (py-crash) [inline docs/basics/beginners-contributing-guide.md:13]: Server or workstation, just say log in your computer or server

File: docs/basics/beginners-contributing-guide.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+
+0. Type `add-apt-repository ppa:git-core/ppa`.
+
+0. Type `apt-get install git` git will be installed.
+
+0. Type `git config --global user.email "email@example.com"` to set an email address that will be used when you post to GitHub.
+
+0. Type `git config --global user.email` to verify the email address.
+
```
Comment (py-crash) [inline docs/basics/beginners-contributing-guide.md:22]: Don't forget to set your name also:
```
git config --global user.name "John Doe"
```
You can also review all your configurations with `git config --list`

File: docs/basics/getting-started.md (hunk 1 above)
Comment (WhiteNight121) [reply docs/basics/getting-started.md:19]: no it should not, im not sure the best way to fix that..

File: docs/basics/beginners-contributing-guide.md (hunk 2 above)
Comment (WhiteNight121) [reply docs/basics/beginners-contributing-guide.md:111]: ok so u saying we should remove that? if so can do

File: docs/basics/getting-started.md (hunk 1 above)
Comment (pagdot) [reply docs/basics/getting-started.md:19]: I could think of two options:
- remove the change in a new commit
- modify the commits and and force push them to rewrite the history (very ugly)

Comment (WhiteNight121) [reply docs/basics/getting-started.md:19]: ok the first one is what i was thinking but i wanted to make sure before i mess something up as im new to git. thanks.

File: docs/basics/beginners-contributing-guide.md (hunk 8 above)
Comment (WhiteNight121) [reply docs/basics/beginners-contributing-guide.md:22]: thanks @py-crash i added the username command but left the other one out as its good info but its not needed to post.

File: docs/basics/beginners-contributing-guide.md (hunk 2 above)
Comment (py-crash) [reply docs/basics/beginners-contributing-guide.md]: I don't understand what do you mean with "watching".  The title should be:
- "Recommended Reading" if they are only articles
- "Recommended Watching" if they are only videos
- "Recommended Reading/Watching" if they are videos and articles

File: docs/basics/beginners-contributing-guide.md (hunk 8 above)
Comment (py-crash) [reply docs/basics/beginners-contributing-guide.md:22]: Don't forget to change `git config --global user.email` for `git config --list` so the user checks his Name and Email in the same command

File: docs/basics/beginners-contributing-guide.md (hunk 2 above)
Comment (py-crash) [reply docs/basics/beginners-contributing-guide.md]: They are good links, and should be there

Comment (WhiteNight121) [reply docs/basics/beginners-contributing-guide.md]: ok ill add them back in

File: docs/basics/beginners-contributing-guide.md (hunk 3 above)
Comment (WhiteNight121) [reply docs/basics/beginners-contributing-guide.md]: done

File: docs/basics/beginners-contributing-guide.md (hunk 4 above)
Comment (WhiteNight121) [reply docs/basics/beginners-contributing-guide.md]: done

File: docs/basics/beginners-contributing-guide.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
+# Beginners Contributing Guide
+
```
Comment (py-crash) [inline docs/basics/beginners-contributing-guide.md:2]: Mention the "Contribution guidelines" that are in CONTRIBUTING.md
Also CONTRIBUTING.md should also mention this article

Comment (WhiteNight121) [reply docs/basics/beginners-contributing-guide.md:2]: @py-crash haha good call ill added it

**Comment by py-crash** [comment]
Great job!

**Comment by WhiteNight121** [comment]
Np glad to help and learn about along the way

File: docs/basics/beginners-contributing-guide.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
+# Beginners Contributing Guide
```
Comment (RealOrangeOne) [inline docs/basics/beginners-contributing-guide.md:1]: This page mirrors a lot of https://github.com/selfhostedshow/wiki/blob/24e0b21ad9801e48db4af280c02d4cf354e93e64/CONTRIBUTING.md#how-to-contribute, so we should probably move the missing content of that in here, and just link to here directly.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -1,5 +1,6 @@
 # Contributing to the Self Hosted Show Wiki
 
+See the [Beginners Contributing Guide](/basics/beginners-contributing-guide/) for step by step instructions on how to add content.
```
Comment (RealOrangeOne) [inline CONTRIBUTING.md:3]: This should be a fully resolved link (domain and all), so it works when viewed on GitHub too!

File: docs/basics/beginners-contributing-guide.md [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+0. Click ![newpulrequest](images/newpullrequrest.PNG) in the top(ish) left, verify the the branch is set to "dev". 
+
+0. Under Compare changes click "compare across forks" link, verify the **base** repository is the [upstream](https://github.com/selfhostedshow/wiki) site and the head repository is your repo.
+
+0. The changes that were made should come automatically, make the comment succinct and meaningful.
+
+0. Submit a pull request by clicking ![createpullrequest](images/createpullrequest.PNG).
+
+0. The commit has now been added to the pull request the Wiki-Admins will review and if approved will get pushed to the master branch for publishing.
+
```
Comment (RealOrangeOne) [inline docs/basics/beginners-contributing-guide.md:101]: Something i'd be interested in here which would be quite useful would be what to do when there's feedback. Mostly in the fact that there's no need to recreate branches / whatever, just push to new ones (and please don't force!)

File: docs/basics/beginners-contributing-guide.md (hunk 10 above)
Comment (WhiteNight121) [reply docs/basics/beginners-contributing-guide.md:1]: idk why the files saved that way with the extensions but easy fix, in terms of move content from and to https://github.com/selfhostedshow/wiki/blob/24e0b21ad9801e48db4af280c02d4cf354e93e64/CONTRIBUTING.md#how-to-contribute, page i might need more info on what we want that to look like

File: CONTRIBUTING.md (hunk 11 above)
Comment (WhiteNight121) [reply CONTRIBUTING.md:3]: also easy fix and ya i was not sure what it needed to be thanks.

File: docs/basics/beginners-contributing-guide.md (hunk 12 above)
Comment (WhiteNight121) [reply docs/basics/beginners-contributing-guide.md:101]: @RealOrangeOne thanks for all the feed back. for this one ya that sounds good to me but again what do we want to do when there is feed back point them to github? also dont no what "(and please don't force!)" means again sorry new to git/github

Comment (RealOrangeOne) [reply docs/basics/beginners-contributing-guide.md:101]: Force pushing basically overrides the commit history for the PR, and makes reviews more annoying. 

When there's feedback, people just commit and push to the same branch they originally submitted, but with changes. And the commit message shouldn't be something generic like "changes from review", because that's not very descriptive. Submitting PRs / updates with multiple commits is completely fine!

File: docs/basics/beginners-contributing-guide.md (hunk 10 above)
Comment (RealOrangeOne) [reply docs/basics/beginners-contributing-guide.md:1]: This page should be specifically how to contribute in terms of making a change to the wiki. The `CONTRIBUTING.md` should be more about what contributions are wanted / high level process for PR / review cycle. Most of that it already has, which is good, but now it's expanded more here.

**Comment by RealOrangeOne** [comment]
Also, I think this page should live under the "wiki" section rather than "basic". "basic" is more for basics of self hosting, rather than about the wiki itself. Wiki stuff should live in "wiki"

File: docs/basics/beginners-contributing-guide.md (hunk 12 above)
Comment (WhiteNight121) [reply docs/basics/beginners-contributing-guide.md:101]: ok i can add that info as a note to that page

**Comment by WhiteNight121** [comment]
> 
> 
> Also, I think this page should live under the "wiki" section rather than "basic". "basic" is more for basics of self hosting, rather than about the wiki itself. Wiki stuff should live in "wiki"

ok i can move it np i was not 100% on where it should go.

File: docs/basics/beginners-contributing-guide.md (hunk 12 above)
Comment (WhiteNight121) [reply docs/basics/beginners-contributing-guide.md:101]: updated and moved to wiki folder

**Comment by BCNelson** [comment]
I think that It is necessary for the beginners guide but we will want to avoid giving explicit commands such as `sudo apt` as it assumes a debian based system.

**Comment by WhiteNight121** [comment]
> 
> 
> I think that It is necessary for the beginners guide but we will want to avoid giving explicit commands such as `sudo apt` as it assumes a debian based system.

i agree but in the per-recs "This guide was written for linux" i could change to debian based and add a link to how to install on other OS if you think that would be good but i think someone should be in that step to help with the overall flow.

**Comment by BCNelson** [comment]
> should i move the new terminology doc to a new PR? if so how do i do that as thats what i was trying to do...

Yes that should be another PR

**Comment by WhiteNight121** [comment]
> 
> 
> > should i move the new terminology doc to a new PR? if so how do i do that as thats what i was trying to do...
> 
> Yes that should be another PR

ya i agree but i guess to do that i would of needed to make a new branch... we were just talking about it in the wiki-chat if its something we want to standardize/document

File: docs/basics/beginners-contributing-guide.md (hunk 10 above)
Comment (WhiteNight121) [reply docs/basics/beginners-contributing-guide.md:1]: ok that works for me i guess i can take a stab at linking and see if thats what you were thinking.

**Comment by IronicBadger** [comment]
@WhiteNight121 what's the status of this? did you come to an agreement on what should live in the document vs not?

**Comment by WhiteNight121** [comment]
so i also think its done i think we did sort it out but again ya this was some time ago...

**Comment by IronicBadger** [comment]
I think this is good to go. Changed references of dev branch to master and removed duplicate entry of beginners contributing guide.
